generic/ras.py

```diff
@@ -40,6 +40,14 @@ def setUp(self):
         architecture = os.uname()[4]
         if "ppc" not in architecture:
             self.skip("supported only on Power platform")
+        if 'PowerNV' in process.system_output('grep ^platform /proc/cpuinfo'):
```

Comment:
Not sure there is not much of difference. 
```
$ time python -c "from avocado.utils import process"

real	0m0.175s
user	0m0.139s
sys	0m0.035s

$ time python -c "from avocado.utils import process; print 'GenuineIntel' in process.system_output('grep GenuineIntel /proc/cpuinfo')"
True

real	0m0.182s
user	0m0.151s
sys	0m0.030s
```
With the difference, both takes same time
```
$ time python -c "print 'GenuineIntel' in open('/proc/cpuinfo', 'r').read()"
True

real	0m0.010s
user	0m0.007s
sys	0m0.003s
```